lab_1/main.py

```diff
@@ -2,41 +2,45 @@
 Lab 1
 A concordance extraction
 """
+text = open('data.txt', encoding = 'utf-8').read()
 
+# ФУНКЦИЯ РАБОТАЕТ, НО ПОСМОТРЕТЬ, ПОЧЕМУ  ТЕСТЫ НЕ ВИДЯТ ФАЙЛ "TOKENS.TXT" + '`the'
+def tokenize(text):
```

Comment:
please, leave the docstrings (description) here (everywhere)
"""
Splits...
"""